index.html

```diff
@@ -176,10 +176,12 @@ <h1 class="contact_title fw500">CONTACT US</h1>
         <p class="contact_text ffinter fw400">If you have an application you are interested in developing, 
           a feature that you need built or a project that needs coding. I'd love to help with it</p>
       </div>
-      <input class="input_text ffinter fw400" type="text" name="title" placeholder="Text Input">
-      <input class="input_email ffinter fw400" type="text" name="email" placeholder="jeremy@gmail.com">
-      <input class="input_text input_msg" type="text" name="msg" placeholder="Enter text here">
-      <button type="button" class="contact_button theme44">See Project</button>
+      <form action="https://formspree.io/f/xpzbozga" method="post">
+        <input class="input_text ffinter fw400" type="text" name="title" placeholder="Text Input" maxlength="30" required>
```

Comment:
Kindly replace placeholder text from `Text Input` to `Full Name` or `John Doe` because it makes more sense what the user should enter here. 👍🏻